packages/mobile/src/firebase/db-rules.json

```diff
@@ -20,7 +20,15 @@
     "registrations": {
       "$address": {
         ".read": "auth.uid != null && $address === root.child('users').child(auth.uid).child('address').val()",
-        ".write": "auth.uid != null && $address === root.child('users').child(auth.uid).child('address').val()"
+        "fcmToken": {
```

Comment:
What's this for?

Reply:
So that users can write their token and language but only read their dailyLimit. Otherwise a user would technically be able to change their dailyLimit if they managed to take the firebase key somehow from the binary

Reply:
All those fields are stored on `registrations/{address}`. Previously, a user could write anything there, so there was a generic `.write` rule. Now, users can write only certain fields there, so there are specific rules for each field.

Reply:
Ok gotcha. Please include in `Other Changes`!

Reply:
This is not really an other change, this was required to make the feature work, we're adding this property